x-pack/plugins/security/server/authorization/privileges/feature_privilege_iterator/feature_privilege_iterator.test.ts

```diff
@@ -0,0 +1,471 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Feature } from '../../../../../features/server';
+import { featurePrivilegeIterator } from './feature_privilege_iterator';
+
+describe('featurePrivilegeIterator', () => {
+  it('handles features with no privileges', () => {
+    const feature = new Feature({
+      id: 'foo',
+      name: 'foo',
+      privileges: 'none',
+      app: [],
+    });
+
+    const actualPrivileges = [];
+    for (const privilege of featurePrivilegeIterator(feature, {
```

Comment:
You can use [Array.from](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) here to make this a bit terser.

Reply:
oh yeah! Thanks for the pointer 👍